### PR TITLE
implementing firebase-cloud-storage

### PR DESCRIPTION
**Pull request by salinda-yasas-99**

### CODE REVIEW

File: src/main/java/com/gmart/gmart_api/GMartApiApplication.java [hunk 1]
```diff
@@ -18,4 +20,9 @@ public ModelMapper modelMapper() {
         return new ModelMapper();
     }
 
+//    @Bean
+//    public MultipartResolver multipartResolver() {
+//        return new StandardServletMultipartResolver();
+//    }
+
```
Comment (nimeshnuwanga) [inline src/main/java/com/gmart/gmart_api/GMartApiApplication.java:27]: remove comments

**Review by nimeshnuwanga — CHANGES_REQUESTED** [review]

**Comment by nimeshnuwanga** [comment]
Could you explain the purpose of creating the pull request if merging is not the objective?

**Comment by salinda-yasas-99** [comment]
> Could you explain the purpose of creating the pull request if merging is not the objective?

gihin paw nodi nida ganim bn

**Review by nimeshnuwanga — APPROVED** [review]

File: src/main/java/com/gmart/gmart_api/controller/ProductController.java [hunk 2]
```diff
@@ -24,8 +25,42 @@ public ProductController(ProductService productService) {
         this.productService = productService;
     }
 
-    @PostMapping
-    public ResponseEntity<GetProductDto> AddNewProduct(@RequestBody ProductDto productDto) {
+    @PostMapping(consumes = {MediaType.MULTIPART_FORM_DATA_VALUE,MediaType.APPLICATION_OCTET_STREAM_VALUE})
+    public ResponseEntity<GetProductDto> addNewProduct(
```
Comment (lakshankd) [inline src/main/java/com/gmart/gmart_api/controller/ProductController.java:29]: Use RequestBody

**Review by lakshankd — CHANGES_REQUESTED** [review]

**Review by lakshankd — APPROVED** [review]

**Pull request merged**